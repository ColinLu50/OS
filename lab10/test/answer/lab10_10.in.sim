3960
410
414
231
410
223
