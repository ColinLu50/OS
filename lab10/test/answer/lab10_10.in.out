FCFS
174 -> 216 -> 229 -> 204 -> 49 -> 71 -> 49 -> 205 -> 240 -> 146 -> 2 -> 222 -> 123 -> 39 -> 39 -> 63 -> 91 -> 34 -> 203 -> 189 -> 201 -> 118 -> 196 -> 59 -> 96 -> 109 -> 172 -> 66 -> 171 -> 214 -> 14 -> 235 -> 204 -> 214 -> 27 -> 226 -> 172 -> 71 -> 130 -> 194 -> 22 -> 44 -> 196 -> 100 -> 142 -> 120 -> 167 -> 6 -> 46
3960
SSTF
174 -> 172 -> 172 -> 171 -> 167 -> 146 -> 142 -> 130 -> 123 -> 120 -> 118 -> 109 -> 100 -> 96 -> 91 -> 71 -> 71 -> 66 -> 63 -> 59 -> 49 -> 49 -> 46 -> 44 -> 39 -> 39 -> 34 -> 27 -> 22 -> 14 -> 6 -> 2 -> 189 -> 194 -> 196 -> 196 -> 201 -> 203 -> 204 -> 204 -> 205 -> 214 -> 214 -> 216 -> 222 -> 226 -> 229 -> 235 -> 240
410
SCAN
174 -> 172 -> 171 -> 167 -> 146 -> 142 -> 130 -> 123 -> 120 -> 118 -> 109 -> 100 -> 96 -> 91 -> 71 -> 66 -> 63 -> 59 -> 49 -> 46 -> 44 -> 39 -> 34 -> 27 -> 22 -> 14 -> 6 -> 2 -> 0 -> 189 -> 194 -> 196 -> 201 -> 203 -> 204 -> 205 -> 214 -> 216 -> 222 -> 226 -> 229 -> 235 -> 240
414
C-SCAN
174 -> 172 -> 171 -> 167 -> 146 -> 142 -> 130 -> 123 -> 120 -> 118 -> 109 -> 100 -> 96 -> 91 -> 71 -> 66 -> 63 -> 59 -> 49 -> 46 -> 44 -> 39 -> 34 -> 27 -> 22 -> 14 -> 6 -> 2 -> 0 -> 246 -> 240 -> 235 -> 229 -> 226 -> 222 -> 216 -> 214 -> 205 -> 204 -> 203 -> 201 -> 196 -> 194 -> 189
231
LOOK
174 -> 172 -> 171 -> 167 -> 146 -> 142 -> 130 -> 123 -> 120 -> 118 -> 109 -> 100 -> 96 -> 91 -> 71 -> 66 -> 63 -> 59 -> 49 -> 46 -> 44 -> 39 -> 34 -> 27 -> 22 -> 14 -> 6 -> 2 -> 189 -> 194 -> 196 -> 201 -> 203 -> 204 -> 205 -> 214 -> 216 -> 222 -> 226 -> 229 -> 235 -> 240
410
C-LOOK
174 -> 172 -> 171 -> 167 -> 146 -> 142 -> 130 -> 123 -> 120 -> 118 -> 109 -> 100 -> 96 -> 91 -> 71 -> 66 -> 63 -> 59 -> 49 -> 46 -> 44 -> 39 -> 34 -> 27 -> 22 -> 14 -> 6 -> 2 -> 240 -> 235 -> 229 -> 226 -> 222 -> 216 -> 214 -> 205 -> 204 -> 203 -> 201 -> 196 -> 194 -> 189
223
